tests/integration_cases/test_raft.rs

```diff
@@ -2730,8 +2728,13 @@ fn test_restore() {
         s.get_metadata().get_term()
     );
     assert_eq!(
-        sm.prs().nodes(),
-        s.get_metadata().get_conf_state().get_nodes()
+        sm.prs().voter_ids().iter().cloned().collect::<HashSet<_>>(),
```

Comment:
Why not just `*sm.prs().voter_ids()`?

Reply:
Seems it was forgotten to be changed many comments ago.